lib/v2/qdu/router.js

```diff
@@ -1,3 +1,4 @@
 module.exports = (router) => {
     router.get('/jwc', require('./jwc'));
+    router.get('/houqin', require('./houqin'));
```

Comment:
Insert new entries in alphabetical order.
```suggestion
    router.get('/houqin', require('./houqin'));
    router.get('/jwc', require('./jwc'));
```